packages/types/src/cache.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Cache interface used by useParserCache and useValidationCache.
+ */
+export interface Cache<T> {
+  get(key: string): T | undefined;
+  set(key: string, value: T): void;
+  clear(): void;
+}
```

Comment:
I think the cache definition should rather be within the packages, instead of the types package. E.g. we also have a cache implementation that differs from this one on the response cache package.

Reply:
Yeah I was not really sure where to put this.
As this interface is used in 2 plugins, which would be the best place for this?

Reply:
I moved the `Cache` interface into the plugins itself in 02188c2.